LayoutTests/pdf/two-pages-continuous-to-discrete-expected.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ UnifiedPDFEnabled=true ] -->
```

Comment:
We enable automatically on platforms where ENABLE_UNIFIED_PDF_FOR_TESTING, so it's ok to drop this. The benefit you get is that you don't get noise from unexpected results coming from test runs on platform (versions) without full support for Unified PDF.